content/docs/user-guide/troubleshooting.md

```diff
@@ -1,6 +1,7 @@
 # Troubleshooting
 
-Here we provide help for some of the problems that DVC user might stumble upon.
+Here we provide help for some of the problems that DVC user might stumble upon,
+based on frequent support questions (FAQ).
```

Comment:
FAQ != Frequent Support questions though. Also, not clear if this change is useful.

Reply:
It was introduced for SEO. We don't really have the term "FAQ" anywhere. Rolled this back for now but should we put it somewhere? Maybe in the User Guide index even.